### PR TITLE
Finish UI 🎉

### PR DESCRIPTION
**Pull request by Takhion**

### CODE REVIEW

File: lib/eventdetails/event.dart [hunk 1]
```diff
@@ -20,14 +27,23 @@ const kEvent = const Event(
     title: "A New Hope",
     speakers: const <Speaker>[
       const Speaker(
-          fullName: "Sebastiano Poggi",
+          fullName: "Eugenio Marletti",
```
Comment (rock3r) [inline lib/eventdetails/event.dart:30]: I see how this is, huh 😝

Comment (rock3r) [reply lib/eventdetails/event.dart:30]: (I totally know it's the same in squanchy)

File: lib/eventdetails/eventdetails.dart [hunk 2]
```diff
@@ -12,80 +13,178 @@ class EventDetailsPage extends StatefulWidget {
...
-              child: new Column(
-                  children: new List.from(speakersChildrenFor(event.speakers))
-                    ..add(titleWidget),
-                  crossAxisAlignment: CrossAxisAlignment.start)),
+    final appBarBottom = new Padding(
+        padding: new EdgeInsets.only(left: 72.0, right: 8.0, bottom: 8.0),
+        child: new Column(
+            mainAxisSize: MainAxisSize.min,
+            children: _speakersWidgets(speakerTextStyle, event.speakers)
+              ..add(titleWidget),
```
Comment (rock3r) [inline lib/eventdetails/eventdetails.dart:63]: Fun fact: I _adore_ the `..` operator.

File: lib/utils/flexible_app_bar_widget.dart [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  @override
+  double get bottomHeight {
+    BuildContext savedContext;
+    final builder = new Builder(
+        builder: (BuildContext context) {
+          savedContext = context;
+          return child;
+        },
+        );
+    new OffscreenWidgetTree(screenSize)
```
Comment (rock3r) [inline lib/utils/flexible_app_bar_widget.dart:29]: ✨ pure magic

File: lib/utils/flexible_app_bar_widget.dart [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+
+import 'offscreen_widget_tree.dart';
+
+class FlexibleAppBarBottomWidget extends AppBarBottomWidget {
+  FlexibleAppBarBottomWidget(this.screenSize, this.child);
+
+  FlexibleAppBarBottomWidget.fromContext(BuildContext context, Widget child)
```
Comment (rock3r) [inline lib/utils/flexible_app_bar_widget.dart:9]: Can you move this above the ctor it uses?

File: lib/utils/offscreen_widget_tree.dart [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  OffscreenWidgetTree(Size size)
+      : renderView = new _OffscreenRenderView(size) {
+    renderView.attach(pipelineOwner);
+    renderView.scheduleInitialFrame();
+  }
+
+  final RenderView renderView;
+  final buildOwner = new BuildOwner();
+  final pipelineOwner = new PipelineOwner();
+  RenderObjectToWidgetElement<RenderBox> root;
```
Comment (rock3r) [inline lib/utils/offscreen_widget_tree.dart]: Whiteline before to separate from finals?

File: lib/utils/offscreen_widget_tree.dart [hunk 6]
```diff
@@ -0,0 +1,45 @@
+/*
+adapted from:
+https://github.com/flutter/flutter/blob/master/packages/flutter/test/widgets/independent_widget_layout_test.dart
+*/
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+
+class OffscreenWidgetTree {
+  OffscreenWidgetTree(Size size)
```
Comment (rock3r) [inline lib/utils/offscreen_widget_tree.dart]: Can we call this `size` something more clear like `maxSize` or something

File: lib/eventdetails/eventdetails.dart (hunk 2 above)
Comment (Takhion) [reply lib/eventdetails/eventdetails.dart:63]: Fun fact: me too!

File: lib/eventdetails/event.dart (hunk 1 above)
Comment (Takhion) [reply lib/eventdetails/event.dart:30]: hehe, yes, sure, that's the reason 😝

File: lib/utils/flexible_app_bar_widget.dart (hunk 3 above)
Comment (Takhion) [reply lib/utils/flexible_app_bar_widget.dart:29]: ✨ 🔮 🌙

File: lib/utils/offscreen_widget_tree.dart (hunk 6 above)
Comment (Takhion) [reply lib/utils/offscreen_widget_tree.dart]: uhm I guess technically it's the `containerSize`

File: lib/utils/flexible_app_bar_widget.dart (hunk 4 above)
Comment (Takhion) [reply lib/utils/flexible_app_bar_widget.dart:9]: I think the primary constructor should always go first

**Pull request merged**